event/event.gen.go

```diff
@@ -125,12 +145,6 @@ func Close(listener func(*vecty.Event)) *vecty.EventListener {
 	return &vecty.EventListener{Name: "close", Listener: listener}
 }
 
-// The compass used to obtain orientation data is in need of calibration.
-// https://developer.mozilla.org/docs/Web/Events/compassneedscalibration
-func CompassNeedsCalibration(listener func(*vecty.Event)) *vecty.EventListener {
```

Comment:
I found it! The issue is that it was removed from the standard events table (because Firefox doesn't support it, but IE 11 does). Removing it seems to be correct behavior, so let's merge as-is and reconsider how to handle non-standard events in the future (will open an issue for this)